.gitignore

```diff
@@ -43,3 +43,6 @@ npm-debug.log
 .firebase
 firebase
 firebase.json
+
+#Editor configs
+.idea
```

Comment:
please remove. If you would like this added, please open a separate issue so that we can discuss with the infra folks.

Reply:
removed